### ISSUE TITLE
How to auto download all missing subtitles ?

### ISSUE THREAD

**Issue by ludoo0d0a** [ocombe/subNode#5]
is there a feature to auto install all missing subtitles ?
with any schedule planner ? (at least, one time per day)

Great job, works perfectly on my synology :)

Bravo ;)

**Comment by bastervrugt** [comment]
+1

**Comment by ocombe** [comment]
Yep, I should restart my work on this project, I have a few ideas of improvement (the auto download & schedule are on the list).
I might be working on this as I try Angular 2, it would be a good test project. But I have so many other projects on the side as well :-/

**Comment by PetitMalin** [comment]
@ludoo0d0a  Have you installed SubNode on your Synology ?!? How to ?!?

**Comment by ludoo0d0a** [comment]
Package nodejs syno (or community)
Install npm
Git clone + pm2
Still failed to run @ startup so manual launch script for now.
Bonne chance petitmalin!

**Comment by PetitMalin** [comment]
Thank's for the reply @ludoo0d0a 
What i have do 

I have already installed Nodejs for SYNO but it's version 0.10.36.
but If you launch the init.sh on setup you have a warning message for 0.8 version required
Npm is already installed on version 1.4.28 (with the Nodejs for SYNO ?!?) 

Git clone + pm2 ?!? (I did not understand )

For Pm2 = installed with a SSH command with putty "npm install pm2 -g"

Finally when i launch a SSH command with putty "node serverWrapper.js" => It works !!! 
but when i closed putty Subnode close too...and loose the config when you launch again.

A little tutorial would be helpful !!! ;) 
Thx for your help

**Comment by ocombe** [comment]
The auto download feature is now on the roadmap for v2.0 (https://github.com/ocombe/subNode/issues/8)
